blocks/identity-block/features/social-sign-on/default.test.jsx

```diff
@@ -21,20 +21,12 @@ describe("Subscriptions Social Login Feature", () => {
 		useIdentity.mockImplementation(() => ({ isInitialized: false }));
 
 		render(<SocialSignOnBlock customFields={defaultCustomFields} />);
-		expect(screen.queryByTestId("social-sign-on-container")).toBeNull();
+		expect(screen.queryByTestId("social-sign-on-container")).toBeNull(); // eslint-disable-line
```

Comment:
@LauraPinilla This line doesn't have any lint error. // eslint-disable-line isn't needed in this line